src/Avalonia.Controls/TreeView.cs

```diff
@@ -168,7 +168,7 @@ public void ExpandSubTree(TreeViewItem item)
 
             var panel = item.Presenter.Panel;
 
-            if (panel != null)
+            if (item.Presenter?.Panel != null)
```

Comment:
Not sure how this works, the line above references `Presenter.Panel` without the null check?